src/index.postcss

```diff
@@ -1,3 +1,36 @@
 @tailwind base;
 @tailwind components;
-@tailwind utilities;
\ No newline at end of file
+@tailwind utilities;
+
+@layer components {
+  .container {
+    @apply bg-unnamed-color-222224 h-screen w-screen max-w-none text-unnamed-color-f7f7f7
+  }
+  .login-form {
+    @apply flex flex-col relative top-1/2 mx-auto w-350 h-200
+  }
+  .credentials {
+    @apply flex flex-col
+  }
+  .email {
+    @apply bg-unnamed-color-292A2B w-350 h-40 mb-4 p-3
+  }
+  .password {
+    @apply bg-unnamed-color-292A2B w-350 h-40 mb-4 p-3
```

Comment:
spójrz na klasę `.email` i `.password`. Jest dokładnie to samo, a chyba po to się robi takie klasy, by się kod nie powtarzał.

Reply:
To trochę czepianie się już, ale klasa` .login-form` i  `.credentials` zawierają się w sobie, tzn. klasa login-form zawiera wszystko, co znajduje się w credentials. Można by tutaj wykorzystać dziedziczenie.